root/ngtemplate/sample/show.tt

```diff
@@ -36,6 +36,11 @@
         <div class="col-md-4 col-sm-6">
             [% INCLUDE 'sample/panels/notes.tt' %]
         </div>
+        [% IF features.isla_worksheet %]
```

Comment:
For reasons not clear to me I don't love that this is a panel of its own, but we don't have another place in which to collect buttons for miscellaneous operations on a sample so I'm fine with it for now.